tests/Directory.Build.targets

```diff
@@ -25,11 +25,15 @@
     <Error Condition="'$(TestArchiveTestsDir)' == ''" Text="TestArchiveTestsDir property to archive the test folder must be set." />
     <PropertyGroup>
       <TestsArchiveSourceDir Condition="'$(TestsArchiveSourceDir)' == ''">$(OutDir)</TestsArchiveSourceDir>
+      <ZipTestArchiveTfm></ZipTestArchiveTfm>
+      <ZipTestArchiveTfm Condition="'$(TargetFramework)' != '$(DefaultTargetFramework)'">-$(TargetFramework)</ZipTestArchiveTfm>
     </PropertyGroup>
 
-    <MakeDir Directories="$(TestArchiveTestsDir)" />
-    <ZipDirectory SourceDirectory="$(TestsArchiveSourceDir)"
-                  DestinationFile="$([MSBuild]::NormalizePath('$(TestArchiveTestsDir)', '$(MSBuildProjectName).zip'))"
+    <!-- TestsArchiveSourceDir is empty (OutDir) for multi-target build step -->
+    <MakeDir Condition="$(TestsArchiveSourceDir) != ''" Directories="$(TestArchiveTestsDir)" />
+    <ZipDirectory Condition="$(TestsArchiveSourceDir) != '' and '$(TargetFramework)' == '$(DefaultTargetFramework)'"
```

Comment:
Also, the `'$(TargetFramework)' == '$(DefaultTargetFramework)'` means that we would get only the default zip file for `net8.0`, and not for `net9.0`.

Reply:
I could use some help for this one, when I ran helix without this it failed because it would not find a trx file matching the net90.zip file (I rephrased the error message).

Reply:
Um I can work on that bit.